x-pack/plugins/maps/public/embeddable/map_embeddable_factory.ts

```diff
@@ -7,23 +7,56 @@
 import _ from 'lodash';
 import { i18n } from '@kbn/i18n';
 import { IIndexPattern } from 'src/plugins/data/public';
-// @ts-ignore
-import { getMapsSavedObjectLoader } from '../angular/services/gis_map_saved_object_loader';
-import { MapEmbeddable, MapEmbeddableInput } from './map_embeddable';
-import { getIndexPatternService, getHttp, getMapsCapabilities } from '../kibana_services';
 import {
   EmbeddableFactoryDefinition,
   IContainer,
 } from '../../../../../src/plugins/embeddable/public';
+import '../index.scss';
```

Comment:
I wonder if we should [lazy load the css here too](https://blog.webjeda.com/lazy-load-css/). That's pretty much all of Maps css there since it's almost all required by embeddables.

Reply:
maybe (?)  I'm trying to keep this minimal in this first step and just focus on the JS dependency tree.